### PR TITLE
Add Alby to all applicable parts of the site

### PR DESCRIPTION
**Pull request by secondl1ght**
Hello, Alby is a lightning browser extension that supports Lightning Addresses! I have added links to Alby to parts of the site. Thanks!!

### CODE REVIEW

File: components/hero.js [hunk 1]
```diff
@@ -253,6 +253,7 @@ export class Hero extends PureComponent {
               <LoopedTextPart>zebedee.io</LoopedTextPart>
               <LoopedTextPart>coinos.io</LoopedTextPart>
               <LoopedTextPart>ln.tips</LoopedTextPart>
+              <LoopedTextPart>getalby.com</LoopedTextPart>
```
Comment (andrerfneves) [inline components/hero.js:256]: So if you connect your node to the Alby browser extension, you automatically get a `getalby.com` domain? I'm not sure that's the case. This piece is to loop through the many domains available.

File: components/providers.js [hunk 2]
```diff
@@ -299,6 +304,13 @@ export const Providers = () => (
           </ImageWrapper>
           <ProviderSignUpButton target="_blank" href="https://t.me/LightningTipBot">Open Telegram</ProviderSignUpButton>
         </ProviderCard>
+        <ProviderCard>
```
Comment (andrerfneves) [inline components/providers.js:307]: Same comment as the one above.

File: components/hero.js (hunk 1 above)
Comment (secondl1ght) [reply components/hero.js:256]: You don't automatically get a domain but with a few steps you can setup your Alby domain.

Comment (secondl1ght) [reply components/hero.js]: I talked to the maintainers of Alby and they agree that getting a @getalby.com lightning address right now is sort of a hidden feature. You have to go to https://github.com/login/ to do it. So I have removed this part from the PR and we can add it in at a later date when the lightning address is more integrated for users.

Comment (secondl1ght) [reply components/hero.js]: I talked to the maintainers of Alby and they agree that getting a @getalby.com lightning address right now is sort of a hidden feature. You have to go to https://github.com/login/ to do it. So I have removed this part from the PR and we can add it in at a later date when the lightning address is more integrated for users.

File: components/providers.js (hunk 2 above)
Comment (secondl1ght) [reply components/providers.js]: Resolved

**Comment by secondl1ght** [comment]
Comments are now resolved, thanks.

**Pull request merged**